### PR TITLE
Added recipe for roasted pork

### PR DESCRIPTION
**Pull request by bentpetersendk**
I have added the recipe for Roasted Pork

Fixed issue #3

### CODE REVIEW

**Review by MaleneRevsbech — APPROVED** [review]
This looks like a very detailed recipe!

**Review by mtiberti — CHANGES_REQUESTED** [review]
Thank you! Can you please specify a brand name for your favourite cider?

**Comment by bentpetersendk** [comment]
> c

Best copy+paste there is...

**Comment by bentpetersendk** [comment]
> This looks like a very detailed recipe!

Best copy+paste I could find... ;-)

**Comment by bentpetersendk** [comment]
> Thank you! Can you please specify a brand name for your favourite cider?

I have added cider to the recipe as well

File: roasted_pork.txt [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Ingredients
+Cider is important, buy loads of Sommersby
```
Comment (mtiberti) [inline roasted_pork.txt:2]: Good choice!

**Review by mtiberti — APPROVED** [review]
Sounds great!

**Pull request merged**